src/listeners/ready.js

```diff
@@ -16,15 +16,33 @@ module.exports = class extends Listener {
     super(context, {
       once: true
     });
+
     this.style = dev ? yellow : blue;
   }
 
   async run() {
+    await this.createSlashCommands();
+
     await this.printBanner();
     this.printStoreDebugInformation();
     this.container.client.user.setActivity(`${this.container.client.options.defaultPrefix}help | ${this.container.client.guilds.cache.size} Servers`);
   }
 
+  async createSlashCommands() {
+    // this function will tell the SlashCommandStore to update the global and guild commands
+    const slashCommandsStore = this.container.stores.get('slashCommands');
+
+    if(slashCommandsStore) {
```

Comment:
If `slashCommandsStore` is loaded, we will initiate the registration of all slashcommands, otherwise this is not needed (just like deploy script, it's optional)